### PR TITLE
Fix bug with empty limit

### PR DESCRIPTION
**Pull request by Kirill**
But in empty slice with len equal 1 in strings.Split
https://github.com/golang/go/issues/39806

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -41,7 +41,10 @@ func main() {
 
 	// Parse users and limit
 	users := strings.Split(userslist, ",")
-	limit := strings.Split(limitslist, ",")
```
Comment (kirill-scherba) [inline main.go:44]: I don't see an error here!
Sure if 'limitslist' is empty the 'strings.Split(limitslist, ",")' return []string slice with one empty string.
But this empty limit checks in main.go:75

if !(len(limit) == 0 || inSlise(repo, limit)) {
  continue
}

So I think we don't need merge this pull request.

**Review by kirill-scherba — CHANGES_REQUESTED** [review]

File: main.go (hunk 1 above)
Comment (Kirill) [reply main.go:44]: Look issue 39806 in Go. I see this bug at me. I don't set limit, but len(limit) == 1, but slice empty.
Windows 10, Go 1.8.0, and 1.8.1

File: main.go [hunk 2]
```diff
@@ -41,7 +41,10 @@ func main() {
...
+	limit := make([]string, 0)
+	if strings.TrimSpace(limitslist) != "" {
```
Comment (kirill-scherba) [inline main.go]: Please change this line:
if len(strings.TrimSpace(limitslist)) != 0 {

File: main.go [hunk 3]
```diff
@@ -73,6 +76,7 @@ func getRepos(dir, user, maxrepo string, limit []string) (repos []string) {
 
 		// All if 'limit' slice empty or if 'repo' exists in 'limit' slice
 		if !(len(limit) == 0 || inSlise(repo, limit)) {
+			log.Printf("skip %s", repo)
```
Comment (kirill-scherba) [inline main.go]: Remove this line.
If you have more than 20-30 repositories but want backup one than you'll see 20-30 messages... 
So remove this line:  log.Printf("skip %s", repo)

File: main.go (hunk 1 above)
Comment (kirill-scherba) [reply main.go:44]: OK :-)
Please look my new comments and fix it
Thank you! :-)

**Review by kirill-scherba — CHANGES_REQUESTED** [review]
See new comments in code

**Review by Kirill — COMMENTED** [review]
Make changes

**Review by kirill-scherba — APPROVED** [review]
Thanks!
Good work!

**Comment by kirill-scherba** [comment]
All done

**Pull request merged**